web/src/pages/api/user/register.ts

```diff
@@ -36,22 +38,12 @@ export default async function Register(req: Request, res: NextApiResponse) {
 
   try {
     // create new user table via prisma
-    const newUser: User = await prisma.user.create({
+    const newUser = await prisma.user.create({
       data: {
         name: name.trim().toLowerCase(),
         email: email.trim(),
         passwordHash,
       },
-    })
-    .then((user) => {
-      return res.status(201).json({
-        userId: user.id,
-      });
-    })
-    .catch(() => {
-      return res.status(500).json({
-        error: "Registration failed!",
-      });
```

Comment:
Why remove this?

Reply:
The try-catch already catches this stuff and we dont want to send any response here yet